File: python/1020.py
idade = int(input())  # em dias

# ano = 365 dias
# mes = 30 dias

anos = idade // 365
mes = idade % 365 // 30
dias = (idade % 365) % 30

print("{} ano(s)".format(anos))
print("{} mes(es)".format(mes))
print("{} dia(s)".format(dias))

File: Java/_1020.java
import java.util.Scanner;
import java.io.IOException;
public class _1020 {
    public static void main(String args[]) throws IOException{
        Scanner sc=new Scanner(System.in);
        int entrada, dias, anos, meses;

        entrada = sc.nextInt();
        anos = entrada / 365;
        entrada = entrada - anos * 365;

        meses = entrada / 30;
        entrada = entrada - meses * 30;

        dias = entrada;

        System.out.printf("%d ano(s)\n", anos);
        System.out.printf("%d mes(es)\n", meses);
        System.out.printf("%d dia(s)\n", dias);
    }

}
